crates/.gitignore

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+.lspconfig.lua
```

Comment:
do we need this in our repository ? seems like something that goes in the user global gitignore ?

Reply:
Huh, I didn't know user ignore files were a thing.
Live and Learn I guess.
Done